app/javascript/packages/rails-i18n-webpack-plugin/extract-keys-webpack-plugin.js

```diff
@@ -42,7 +42,9 @@ function getAdditionalAssetFilename(filename, locale) {
  * @return {string[]} Translation keys.
  */
 const getTranslationKeys = (source) =>
-  Array.from(source.matchAll(TRANSLATE_CALL)).map(([, key]) => key);
+  Array.from(source.matchAll(TRANSLATE_CALL)).flatMap(([, keys]) =>
+    keys.split(',').map((key) => key.replace(/[ '"\[\]]/g, '')),
+  );
```

Comment:
This logic has started to become rather opaque, I realize. I was pretty close to pulling in a proper JavaScript parser + AST walker, but it still felt like overkill, especially considering it's still rather complex to handle some of the edge cases we deal with here.

The main idea is that the output of the regular expression would split out something like `["foo", "bar", "baz"]` or `"foo"`, then we split that result and eliminate any of the `[`, `]`, quotes, and whitespace to produce an array of keys.